src/solvers/dgsem_p4est/containers.jl

```diff
@@ -108,21 +108,21 @@ end
 
 mutable struct P4estInterfaceContainer{NDIMS, uEltype<:Real, NDIMSP2} <: AbstractContainer
   u             ::Array{uEltype, NDIMSP2}       # [primary/secondary, variable, i, j, interface]
-  element_ids   ::Matrix{Int}                   # [primary/secondary, interface]
+  neighbor_ids  ::Matrix{Int}                   # [primary/secondary, interface]
```

Comment:
Is this change cleared with @efaulhaber? Sorry, I didn't follow all discussions before. The change seems to make sense, though.

Reply:
Yes, my suggestion, actually. I'll see that I can do a full review tomorrow, though.